src/Configurations/PageRulesActions.php

```diff
@@ -98,38 +82,30 @@ public function setCacheOnCookie(bool $value)
             throw new ConfigurationsException("Invalid cookie string.");
         }
 
-        $object = new \stdClass();
-        $object->id = "cache_on_cookie";
-        $object->value = $value;
```

Comment:
This is a `bool` but get passed in raw unlike the on/off conversation, is this correct?